styles/components.css

```diff
@@ -39,4 +39,17 @@
 
 .additional-infos{
   display: none;
+}
+
+
+/* Overview
+###########################################################################  */
+.overview{
+  display: grid;
+  padding: var(--m);
+  margin: 0;
+
+  gap: var(--m);
+  grid-template-columns: repeat(auto-fit, minmax(calc(var(--m)), 1fr));
```

Comment:
Rein syntaktisch alles fine.
Semantisch fällt mir was auf:
- die `calc()`-Funktion ist hier nicht notwendig, da du ja nix berechnest. Außer da kommt noch was :D
- `var(--m)` ist ein sehr kleiner Threshold. Vielleicht was größeres nehmen. Ist ja dann die minimale Breite einer Karte. Hier macht es auch von der Schrift auszugehen. also 20 Zeichen in der Schriftgrößte M:

```css
grid-template-columns: repeat(auto-fit, minmax(calc(var(--m) * 20), 1fr))
```